docs/source/release_notes/release-notes.rst

```diff
@@ -48,8 +48,10 @@ develop
     *    - Type
          - Change
 
-    *    -
-         -
+    *    - |fixed|
+         - kvs.getRanges now also logs ``kvs-slow-log`` if the backing store is being slow to return the requested values.
```

Comment:
nit: probably `getRange` is the one people care about

Reply:
:ack: